### DISCUSSION TITLE
Ajouter des sources à chaque bonne pratique

### DISCUSSION THREAD

**Discussion by devcrafting** [cnumr/best-practices#71] [General]
L'idée est que pour garder les BP courtes mais ne pas laisser sur sa fin le lecteur, on pourrait proposer des sources (articles pour aller plus loin).

**Comment by Trankilloman** [comment]
En effet, cela semble pertinent, cela pourrait passer par des exemples MAIS AUSSI par des contres exemples.

**Comment by devcrafting** [comment]
A noter qu'il serait peut-être bon d'utiliser des liens permanents qu'on maîtrise permettant de lancer un contrôle des sources réguliers pour éviter les 404 ou l'obsolescence, voire même pouvoir changer de source.

**Comment by ddebarbieux** [comment]
Utiliser des ressources extérieures est aussi un moyen de donner plus de poids à une bonne pratique.
